site/docs/v1/tech/tutorials/switch-search-engines.adoc

```diff
@@ -0,0 +1,89 @@
+
+This tutorial walks you through changing your search engine. Reasons for modifying it vary; you may be simplifying your deployment strategy by using the database search engine. Or perhaps you need the advanced searching options of Elasticsearch. Either way, it's an easy switch.
+
+=== Prerequisites
+
+This tutorial assumes you have both the application and the search services running on Linux. You also must be running at least version 1.16.0. 
+
+If you don't, head on over to the link:/docs/v1/tech/installation-guide/fusionauth-search[installation guide]. If you already have the application running, but need to install the search service, link:/docs/v1/tech/installation-guide/fusionauth-search[here are instructions].
+
+If you are running FusionAuth via Docker, a Windows service, or some other mechanism, you'll have to stop and start FusionAuth in an installation specific manner.
+
+=== From Elasticsearch to Database
+
+Stop FusionAuth:
+
+```
+sudo service fusionauth-app stop
+sudo service fusionauth-search stop
+```
+
+Locate your FusionAuth configuration file, it might be at a location similar to `/usr/local/fusionauth/config/fusionauth.properties`.
+
+Comment out or remove the `fusionauth-search.*` properties and the `fusionauth-app.search-servers` value. 
+
+Add or update the `fusionauth-app.search-engine-type` key to have the value `database`.
+```
+fusionauth-app.search-engine-type=database
+```
+
+Restart the FusionAuth application:
+
+```
+sudo service fusionauth-app start
+```
+
+Log into FusionAuth. Navigate to [breadcrumb]#User# and search for a user by email address or first name to confirm that the search engine is working. 
+
+image::tutorials/database-user-search.png[User search with the database search engine,width=1200,role=shadowed]
+
+=== From Database to Elasticsearch 
+
+
+Stop FusionAuth:
+
+```
+sudo service fusionauth-app stop
+```
+
+Locate your config file, it might be at a location similar to `/usr/local/fusionauth/config/fusionauth.properties`.
+
+Uncomment or add the `fusionauth-app.search-servers` value. This must point to your Elasticsearch servers. 
+
+```
+fusionauth-app.search-servers=http://localhost:9021
+```
+
+Add or uncomment and update additional `fusionauth-search.*` properties, as documented in the link:../reference/configuration[configuration reference]. In addition, add or update the `fusionauth-app.search-engine-type` key to have the value `elasticsearch`.
+
+```
+fusionauth-app.search-engine-type=elasticsearch
+```
+
+Restart the FusionAuth application, making sure to start the `fusionauth-search` service as well.
+
+```
+sudo service fusionauth-search start
+sudo service fusionauth-app start
+```
+
+Log in to FusionAuth. Reindex the search index by navigating to [breadcrumb]#System -> Reindex#. This ensures that Elasticsearch has the latest user data. This may take some time, depending on the size of your data.
```

Comment:
Because we are going from db to Elasticsearch, I would phrase this more as "This will initialize the search index and synchronize it with the existing user db" or something along those lines. 


> This may take some time, depending on the size of your data.

Maybe "The reindex request may vary depending upon the number of users and the amount of custom data that requires indexing." ? Something more related to "users" than "data" - while more/less the same, perhaps it is something the end user can relate to better since they will have an idea of how many users they have.